test/unit/server_test.js

```diff
@@ -1,239 +1,290 @@
-describe("server.js tests", function () {
-  beforeEach(function () {
-    this.server = new StellarSdk.Server('https://horizon-live.stellar.org:1337');
+describe('server.js tests', function() {
+  beforeEach(function() {
+    this.server = new StellarSdk.Server(
+      'https://horizon-live.stellar.org:1337'
+    );
     this.axiosMock = sinon.mock(axios);
     StellarSdk.Config.setDefault();
     StellarSdk.Network.useTestNetwork();
   });
 
-  afterEach(function () {
+  afterEach(function() {
     this.axiosMock.verify();
     this.axiosMock.restore();
   });
 
-  describe('Server.constructor', function () {
-    it("throws error for insecure server", function () {
-      expect(() => new StellarSdk.Server('http://horizon-live.stellar.org:1337')).to.throw(/Cannot connect to insecure horizon server/);
+  describe('Server.constructor', function() {
+    it('throws error for insecure server', function() {
+      expect(
+        () => new StellarSdk.Server('http://horizon-live.stellar.org:1337')
+      ).to.throw(/Cannot connect to insecure horizon server/);
     });
 
-    it("allow insecure server when opts.allowHttp flag is set", function () {
-      expect(() => new StellarSdk.Server('http://horizon-live.stellar.org:1337', {allowHttp: true})).to.not.throw();
+    it('allow insecure server when opts.allowHttp flag is set', function() {
+      expect(
+        () =>
+          new StellarSdk.Server('http://horizon-live.stellar.org:1337', {
+            allowHttp: true
+          })
+      ).to.not.throw();
     });
 
-    it("allow insecure server when global Config.allowHttp flag is set", function () {
+    it('allow insecure server when global Config.allowHttp flag is set', function() {
       StellarSdk.Config.setAllowHttp(true);
-      expect(() => new StellarSdk.Server('http://horizon-live.stellar.org:1337')).to.not.throw();
+      expect(
+        () => new StellarSdk.Server('http://horizon-live.stellar.org:1337')
+      ).to.not.throw();
     });
   });
 
-  describe('Server.loadAccount', function () {
+  describe('Server.loadAccount', function() {
     let accountResponse = {
-      "_links": {
-        "self": {
-          "href": "https://horizon-testnet.stellar.org/accounts/GBAH7FQMC3CZJ4WD6GE7G7YXCIU36LC2IHXQ7D5MQAUO4PODOWIVLSFS"
+      _links: {
+        self: {
+          href:
```

Comment:
These responses are copied directly from Horizon. We should ignore linter errors and leave them as are. When these responses change we'll need to manually (?) change them to correct form.